### ISSUE TITLE
Feature extraction for IF images

### ISSUE THREAD

**Issue by pakiessling** [kpcoleman/miso#3]
Hi, congratulation on your publication.

In the paper you mention:
`Image features used as input for each method were extracted from an immunofluorescence image with 3 channels (DAPI, Vimentin, and PCNA) using a pre-trained InceptionV3 model. Patches corresponding to the omics spots were extracted from the immunofluorescence image, resized to 299×299 pixels, and normalized prior to extracting features for each patch using InceptionV3`

Is this just https://pytorch.org/vision/main/models/generated/torchvision.models.inception_v3.html#torchvision.models.inception_v3
or did you use a model trained on microscopy images?
Thank you!

**Comment by kpcoleman** [comment]
Hi,

Thank you for your question. We used the Inception V3 model provided by torchvision, not one trained or fine-tuned on microscopy images.

Best,
Kyle

**Comment by yuequnwang** [comment]
Hi, may I ask if you happen to know where the stained E13 tissue images （in fig4） can be obtained from?
